pandas/tests/arrays/test_integer.py

```diff
@@ -1074,6 +1074,17 @@ def test_cut(bins, right, include_lowest):
     tm.assert_categorical_equal(result, expected)
 
 
+@pytest.mark.parametrize("q", [2, 5, 10])
```

Comment:
can you move to pandas/tests/reshape/test_qcut.py

we group tests by function if possible